packages/enzyme-test-suite/test/enzyme-adapter-react-install-spec.js

```diff
@@ -0,0 +1,86 @@
+import { expect } from 'chai';
+import getAdapterForReactVersion from '../../enzyme-adapter-react-helper/src/getAdapterForReactVersion';
```

Comment:
Feel like I should've been able to do 

```
import getAdapterForReactVersion from 'enzyme-adapter-react-helper/src/getAdapterForReactVersion';
```

Reply:
i think if you add `enzyme-adapter-react-helper` as a dev dep of `enzyme-test-suite`, that this will work?

Reply:
Oh, that would make sense